doc/modules/mod_event_pusher_http.md

```diff
@@ -44,6 +44,20 @@ This module uses a connection pool created by mongoose_http_client. It must be d
 
 Notifications will be POSTed to `http://localhost:8000/webservice/notifications`.
 
+## Payload format
+The HTTP event pusher sends a POST request with Content-Type `application/x-www-form-urlencoded`. The form has the following fields:
+* `author`: lower case username of the user who authored the message
```

Comment:
These fields are not only converted to lower case but actually are processed by specific profiles of "stringprep", as per https://tools.ietf.org/html/rfc6122. You may mention lower case conversion as one of the consequences of this process of course.